src/chi/components/steps/steps.scss

```diff
@@ -87,5 +92,131 @@
         color: $brand-color;
       }
     }
+
+    &.-horizontalLabels {
+      background-color: set-color(white);
+      counter-reset: step;
+      padding: 1rem;
+
+      & > li {
+        &::before {
+          background-color: set-color(white);
+          border: 0.125rem solid $brand-color;
+          content: counter(step);
+          counter-increment: step;
+          font-size: 0.8125rem;
+          height: 1.3rem;
+          margin: 0 auto;
+          width: 1.3rem;
+        }
+
+        &::after {
+          top: 0.5625rem;
+          width: 90%;
+        }
+
+        &.-active {
+          &::before {
+            box-shadow: 0 0 0 0.125rem set-color(white);
+          }
+
+          & ~ li {
+            &::before {
+              background-color: set-color(white);
+              border-color: set-color(grey, 40);
+            }
+
+            & a {
+              color: set-color($brand-color);
+            }
+          }
+
+          & a {
+            margin-left: 0.75rem;
+          }
+        }
+
+        &.-completed {
+          &::before {
+            border: .125rem solid set-color($brand-color);
```

Comment:
@davidojedacl remove this property as it is duplicated from .-horizontalLabels li:before